sklearn/_config.py

```diff
@@ -36,6 +36,8 @@ def set_config(assume_finite=None, working_memory=None):
         to this number of MiB (per job when parallelised), often saving both
         computation time and memory on expensive operations that can be
         performed in chunks. Global default: 1024.
+
+    .. versionadded:: 0.19
```

Comment:
It was in 0.20.

Reply:
It makes more sense to document each parameter than the overall function.

Reply:
> It makes more sense to document each parameter than the overall function.

Honestly this is exactly what I prefer, not sure whether Andreas will agree with us. @jnothman should I push a commit or let's leave it as it is?

Reply:
I guess I would do both, but yes, parameters seems more important.